src/TypeDefinition.js

```diff
@@ -381,6 +381,7 @@ export type NavigationTransitionProps = {
   // is the index of the scene
   scene: NavigationScene,
   index: number,
+  navigation: NavigationScreenProp<NavigationRoute, NavigationAction>,
```

Comment:
I haven't used NavigationExperimental a lot so I might be missing a bigger picture in few cases, esp. when compatibility is concerned. However, thanks for the reminder, I'll check how's it implemented a bit more in deep to keep that in mind for the future.